manifests/service.pp

```diff
@@ -55,7 +55,22 @@
       enable => true,
     }
 
-  } else {
+  } elsif ($::operatingsystem == 'RedHat') or ($::operatingsystem == 'CentOS') {
```

Comment:
RedHat 6 also does not use systemd. Please also do a version check for redhat also